.goreleaser.yml

```diff
@@ -14,39 +12,127 @@ builds:
     goos:
       - darwin
     env:
+      - CGO_ENABLED=1
       - CC=/home/runner/work/osxcross/target/target/bin/o64-clang
       - CXX=/home/runner/work/osxcross/target/target/bin/o64-clang++
+      - LD_LIBRARY_PATH=/home/runner/work/osxcross/target/binaries/lib
     flags:
-      - -tags="netgo ledger"
-      - -trimpath
       - -mod=readonly
+      - -tags="netgo ledger muslc"
+      - -trimpath
     ldflags:
-      - -s -w -X github.com/cosmos/cosmos-sdk/version.Name=regen -X github.com/cosmos/cosmos-sdk/version.AppName=regen -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}} -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}} -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger"
+      - -s -w
+      - -X github.com/cosmos/cosmos-sdk/version.Name=regen
+      - -X github.com/cosmos/cosmos-sdk/version.AppName=regen
+      - -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}}
+      - -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}}
+      - -X github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger,muslc
+      - -linkmode=external -extldflags "-L/home/runner/work -lwasmvm_muslc -Wl"
+
+#  - id: regen-darwin-arm64
```

Comment:
Arm builds are not supported with CosmWasm but this should be set up to work when we update to Cosmos SDK v0.46 and disable CosmWasm.